site/app/controllers/OfficeHoursQueueController.php

```diff
@@ -32,11 +32,21 @@ public function showQueue() {
             );
         }
 
+        $queries = array();
+        parse_str($_SERVER['QUERY_STRING'], $queries);
```

Comment:
You don't need to parse the query string yourself. Adding a `full_history` argument in the function signature is fine if all you need is to parse `/office_hours_queue?full_history=true`. Please have a look at https://submitty.org/developer/router#route-with-parameters